moveos/moveos-stdlib/moveos-stdlib/sources/object.move

```diff
@@ -28,10 +28,21 @@ module moveos_std::object {
         // The value must be the last field
         value: T,
     }
+  
+    /// An object ID
+    struct ObjectID has store, copy, drop {
+        // TODO should use u256 to replace address?
+        id: address,
+    }
+
+    /// Generate a new ObjectID from an address
+    public(friend) fun address_to_object_id(address: address): ObjectID {
+        ObjectID { id: address }
+    }
 
     /// Create a new object, the object is owned by `owner`
     public(friend) fun new<T: key>(ctx: &mut TxContext, owner: address, value: T): Object<T> {
```

Comment:
After having `Context`, should all context-related parameters in functions should be `Context` instead of `TxContext` or `StorageContext` ？